src/System.CommandLine.Tests/Binding/ModelBinderConstructorTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine.Binding;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+using common = System.CommandLine.Tests.Binding.ModelBindingCommandHandlerTests;
+
+namespace System.CommandLine.Tests.Binding
+{
+    public class ModelBinderConstructorTests
```

Comment:
I see two tests in this class for option arguments but none for command arguments.